staging/src/k8s.io/apiserver/pkg/apis/cel/config.go

```diff
@@ -0,0 +1,37 @@
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cel
+
+import apiservercel "k8s.io/apiserver/pkg/cel"
+
+const (
+	// PerCallLimit specify the actual cost limit per CEL validation call
+	// current PerCallLimit gives roughly 0.1 second for each expression validation call
+	PerCallLimit = 1000000
+
+	// RuntimeCELCostBudget is the overall cost budget for runtime CEL validation cost per ValidatingAdmissionPolicy or CustomResource
```

Comment:
Clarify that this "per binding" not "per policy"?  I was glad to see in `controller.go` that `RuntimeCELCostBudget` is passed to each invocation of `bindingInfo.validator.Validate()` for each policy, because this will have far safer and more predictable behavior than forcing the bindings to share a budget.

Reply:
Since a binding can only bind one policy so I guess the budget is also kind of for all validations in that policy? But I can make it more clear

Reply:
Yes, my point was more that there can be many bindings for a policy, so the way the comment is currently written, it sounds like it is describing a "per policy" limit which would be very different than a "per binding" limit (which is what we have).